website/blog/2023-01-31-ferretdb-v-0-9-0-developer-preview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "FerretDB v0.9.0 - Developer Preview"
+slug: ferretdb-v-0-9-0-developer-preview
+author: Alexander Fashakin
+description: "FerretDB 0.9.0 brings with it amazing new features, especially the initial support for aggregation pipelines."
+image: /img/blog/developer_preview.png
+tags: [release]
+date: 2023-01-31
+---
+
+FerretDB 0.9.0 brings with it amazing new features, especially the initial support for aggregation pipelines.
+
+![FerretDB v0.9.0 - Developer Preview](/img/blog/developer_preview.png)
+
+<!--truncate-->
+
+We just rolled out our first Developer Preview - FerretDB v0.9.0, and we are so excited to show you all the exciting updates now available for you.
+This version of [FerretDB](https://www.ferretdb.io) brings us closer to our mission of providing you with a true open-source alternative to MongoDB that supports more real-world use cases.
+
+All this wouldn't be possible without the support of our community and everyone who contributed immensely to this release through their feedback, code, and support.
+Special appreciation goes to [@jkoenig134](https://github.com/jkoenig134) for their first contribution to FerretDB.
+
+## What's new
+
+In this Developer Preview, we've added initial support for aggregation pipelines.
+Currently, only the `$match` and `$count` stages have been implemented.
+
+This `$match` stage is used to filter documents that match specified query conditions.
+
+```js
+db.collection.aggregate([
+    {
+        $match: {
+            <query>
+        }
+    }
+])
+```
+
+On the other hand, the `$count` stage returns the number of documents in the collection that match the specified query conditions.
+
+```js
+db.collection.aggregate([
+    {
+        $count: {
+            <query>
+        }
+    }
+])
+```
```

Comment:
That's incorrect: `$count` does not accept a query condition. See https://www.mongodb.com/docs/v6.0/reference/operator/aggregation/count/

Reply:
PTAL